### PR TITLE
Docs: fix table demo typing error

### PR DESCRIPTION
**Pull request by iicdii**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution

I fixed a typing error defaultFilterValues -> defaultFilteredValue

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | ✅ fix table demo typing error |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

- [X] Doc is updated/provided or not needed
- [X] Demo is updated/provided or not needed
- [X] TypeScript definition is updated/provided or not needed
- [X] Changelog is provided or not needed

-----
[View rendered components/table/demo/head.md](https://github.com/iicdii/ant-design/blob/master/components/table/demo/head.md)

### CODE REVIEW

**Comment by afc163** [comment]
Please run `npm test -- -u` to update snapshot.

File: components/table/demo/head.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ const columns = [
         ],
       },
     ],
-    defaultFilterValues: ['Jim'],
+    defaultFilteredValue: ['Jim'],
```
Comment (afc163) [inline components/table/demo/head.md]: 看了一下例子，这个属性可能是之前调试遗留下来的。这个例子只有四行数据，默认还是都展示出来对用户比较好理解，可以把这一行删掉。

**Comment by yoyo837** [comment]
这个功能4.0的 PR #20088

File: components/table/demo/head.md (hunk 1 above)
Comment (iicdii) [reply components/table/demo/head.md]: @afc163 I agree. I will remove this line and make PR

**Pull request merged**